DB/src/main/java/io/deephaven/db/v2/select/python/ConditionFilterPython.java

```diff
@@ -18,6 +19,8 @@
  */
 public class ConditionFilterPython extends AbstractConditionFilter {
 
+    private Filter filter;
+
```

Comment:
For example, nothing ever reads this variable. Do we intend that getFilter should return it, if not null?